PlottingScripts/Basic_Plot.py

```diff
@@ -0,0 +1,128 @@
+"""This is a basic script for plotting an analysis file."""
+from collections import OrderedDict
+
+import numpy as np
+import pandas as pd
+import pdb
+
+from bokeh.plotting import figure, show, output_file
+
+
+"""This script is just the first step in visualizing the data set. This file
+comprises of the method creating an html bokeh plot and a command at the end to
+implement the method
+"""
+
+
```

Comment:
You can add default values to cols `=['S1', 'ST']` because that will almost always be how we plot it and it will make plotting function calls shorter.